golang/x/relay/keeper/requests.go

```diff
@@ -11,8 +11,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-func (k Keeper) emitProofRequest(ctx sdk.Context, pays, spends []byte, paysValue uint64, id types.RequestID) {
-	ctx.EventManager().EmitEvent(types.NewProofRequestEvent(pays, spends, paysValue, id))
+func (k Keeper) emitProofRequest(ctx sdk.Context, pays, spends []byte, paysValue uint64, id types.RequestID, origin types.Origin) {
```

Comment:
event doesn't display all params. Wondering whether these guys merit the importance of being event published. (for now only Origin)